Dockerfile

```diff
@@ -35,4 +35,9 @@ COPY --from=sdrs-build ./target/storage-sdrs-jar-with-dependencies.jar ./app.jar
 EXPOSE 8080
 
 # Start server on 0.0.0.0
-CMD ["java", "-jar", "./app.jar", "0.0.0.0", "-Xms256m", "-Xmx512m"]
+# CMD ["java", "-jar", "./app.jar", "0.0.0.0", "-Xms256m", "-Xmx512m"]
```

Comment:
we can remove "0.0.0.0" as an argument. The host for web server is set in the applicaitonConfig.xml file now.